client/lib/plans/index.js

```diff
@@ -204,3 +215,33 @@ export function plansLink( url, site, intervalType ) {
 
 	return url;
 }
+
+export function applyTestFiltersToPlansList( planName ) {
+	const filteredPlanConstantObj = Object.assign( {}, plansList[ planName ] );
+	let filteredPlanFeaturesConstantObj = getPlanFeaturesConstantObj( plansList[ planName ].getFeatures() );
+
+	if ( ! isWordadsInstantActivationEnabled() ) {
+		filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
+			( _, key ) => key !== WORDADS_INSTANT
+		);
+	} else if ( planName === PLAN_PREMIUM ) {
+		filteredPlanConstantObj.getDescription = plansList[ planName ].getDescriptionWithWordAdsInstantActivation;
+	}
+
+	if ( ! isGoogleVouchersEnabled() ) {
+		filteredPlanFeaturesConstantObj = pickBy( filteredPlanFeaturesConstantObj,
+			( _, key ) => key !== FEATURE_GOOGLE_AD_CREDITS
+		);
+	} else if ( planName === PLAN_PREMIUM ) {
+		filteredPlanConstantObj.getDescription = plansList[ planName ].getDescriptionWithGoogleVouchers;
```

Comment:
That's a problem, however. There's a reason why I did not do that in the first place:

![selection_032](https://cloud.githubusercontent.com/assets/4988512/16630450/bc612248-43ba-11e6-96a1-71730943a168.png)

I could have simply imported the abtest module and do this also for the features, like 
```js
		getFeatures: () => [ // pay attention to ordering, it is used on /plan page
			FEATURE_FREE_SITE,
			FEATURE_CUSTOM_DOMAIN,
			FEATURE_FREE_THEMES,
			FEATURE_13GB_STORAGE,
			FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
			FEATURE_NO_ADS,
			FEATURE_CUSTOM_DESIGN,
			FEATURE_VIDEO_UPLOADS,
			abtest('googleVouchers') ? FEATURE_GOOGLE_AD_CREDITS : null,
			WORDADS_INSTANT
		],